pantheon-bundle/frontend/src/app/assemblyDisplay.tsx

```diff
@@ -253,8 +253,8 @@ class AssemblyDisplay extends Component<any, any, any> {
     }
 
     private onPublishEvent = () => {
-          this.fetchModuleDetails(this.props)
-     }
+        this.fetchModuleDetails(this.props)
```

Comment:
Although, not a part of this ticket, but as an additional change, can you please rename this method to "fetchAssemblyDetails" . It will sync the process.